go.mod

```diff
@@ -24,3 +25,5 @@ require (
 	golang.org/x/sys v0.0.0-20211116061358-0a5406a5449c
 	google.golang.org/protobuf v1.27.1
 )
+
+replace github.com/opencontainers/runtime-spec/specs-go => ./staging/src/github.com/opencontainers/runtime-spec/specs-go
```

Comment:
Please open PR for runtime-spec

Reply:
ok

Reply:
@AkihiroSuda https://github.com/opencontainers/runtime-spec/pull/1143